sycl/test-e2e/AddressSanitizer/out-of-bounds/buffer/buffer.cpp

```diff
@@ -0,0 +1,34 @@
+// REQUIRES: linux, cpu
+// RUN: %{build} %device_asan_flags -O0 -g -o %t.out
+// RUN: %force_device_asan_rt %{run} not %t.out 2>&1 | FileCheck %s
```

Comment:
why does it use "force_device_asan_rt" here?

Reply:
Removed..